.github/workflows/actions/test-angular-e2e/action.yml

```diff
@@ -29,7 +29,7 @@ runs:
       shell: bash
       working-directory: ./angular/test
     - name: Install Dependencies
-      run: npm install
+      run: npm install --legacy-peer-deps
```

Comment:
Need to create a ticket to track swapping this once stable versions of v16 dependencies land. Alternatively we could continue to allow legacy peer dependencies for the test apps. The only advantage I see to not allowing it currently, is if some package we are using is incompatible with the version of Angular we are testing with. Our tests _should_ catch this scenario however. Unfortunately it seems any time we try to add the next major version test app before official release, we have to change this.

Reply:
Created FW-4092 to track.